src/block_library_controller.js

```diff
@@ -319,3 +324,61 @@ BlockLibraryController.prototype.updateButtons = function(savedChanges) {
   var isInLibrary = this.has(blockType);
   this.view.updateButtons(blockType, isInLibrary, savedChanges);
 };
+
+/**
+ * Return a string of all blockTypes for building a library JSON
```

Comment:
Relevant history of the internet: JSON stands for JavaScript Object Notation and is just a fancy way of saying it's the String representation of a JavaScript object. So usually we leave it as an object until it needs to leave JS.